hope-for-all/src/main/java/comp321/hope_for_all/activities/Message.java

```diff
@@ -176,12 +178,16 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                     case R.id.homeNav:
                         startActivity(new Intent(getApplicationContext(), MainActivity.class));
                         overridePendingTransition(0, 0);
+                        finish();
                         return true;
+
                     case R.id.messageNav:
-                        return false;
```

Comment:
@hailey0882 I hope you dont mind me creating this changes on your code. I noticed that app crashes when nav bottom is pressed from Message back to Profile or Home. Please let me know if that this is ok before merging to main.

Reply:
Yes! It's okay! Thanks for finding this crash ;)

Reply:
Thanks! Merging this to main now.